src/modules/PublicLab.TitleModule.Related.js

```diff
@@ -25,59 +25,142 @@ Results should be in following JSON format:
  */
 module.exports = function relatedNodes(module) {
   var relatedEl;
+  var addedRelatedEl;
+  var addedRelatedPost = 'addedTitles';
```

Comment:
Why string? Use an array. 
If we use string, it will cause bug when id is two digit or more, right? 
"123" matches 12 and 23